packages/venia-ui/lib/components/Header/__tests__/header.spec.js

```diff
@@ -32,6 +34,17 @@ jest.mock('@magento/peregrine/lib/talons/Header/useHeader', () => {
     };
 });
 
+jest.mock('@magento/peregrine', () => {
+    const useWindowSize = jest.fn(() => ({
+        isDesktop: mockIsDesktop
+    }));
+
+    return {
+        ...jest.requireActual('@magento/peregrine'),
```

Comment:
No need to import actual since we're only importing `useWindowSize`

Reply:
@JustinConabreeAbsolunet We do use `createTestInstance` in the test file, so without extending it, we won't have access to `createTestInstance`.

For the performance, you have a good point.
Do we want to impact the user experience for a slightly better Lighthouse score?

Reply:
@mikhaelbois right! I missed that. All good then.

I think we're okay with how you've done it. When they expand, it would show the first _x_ items. Clicking the 'show more' may then be a bit slow, but I think for improved Lighthouse score we're okay. The sub-items aren't very complex so it shouldn't be too bad of a performance hit when they render. QA will just need to check that the form behaves okay with adding/removing fields.